packages/@react-aria/slider/src/useSlider.ts

```diff
@@ -112,7 +112,7 @@ export function useSlider(
       }
     }, draggableProps),
     thumbProps: {
-      labelId: (labelProps.id ?? fieldProps.id)!
+      'aria-labelledby': (labelProps.id ?? fieldProps.id)!
```

Comment:
Hmm, isn't it possible for the thumb to have its own `aria-labelledby`?  (For example, a thumb with a `label`).  In that case, the range input's `aria-labelledby` needs to be composed with the group id plus the thumb's `aria-labelledby`...  I think this might drop that information?

Reply:
That's true. You'd need to merge them. The main thing that I'd prefer to avoid is `labelId` showing up in the API for the thumb since it's an internal option. We'd probably hide this in a context for example, where the slider provides the thumb props and the slider state and the thumb consumes them. So I guess it would be up to the component to combine the props from context with the ones from its own props.

Will need to think about this more...

Reply:
Ended up following the pattern we've used in other components for passing ids between a parent and child: attaching to the state object via a weakmap. This way the calling component doesn't need to worry about passing the thumb props through at all. If hooks supported writing to context, we'd use that, but attaching the data to the state is the easiest way for now. This is currently used in calendar, listbox, tabs, and a few others.